packages/default/scss/checkbox/_layout.scss

```diff
@@ -43,21 +43,24 @@
     }
 
     // Hover state
-    .k-checkbox:hover {
+    .k-checkbox:hover,
+    .k-checkbox.k-state-hover {
```

Comment:
The checkbox can always have a cursor poniter :) After all, we see it only when we hover it.